specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/gallery.json

```diff
@@ -1069,6 +1069,20 @@
       ],
       "description": "Describes the basic gallery artifact publishing profile."
     },
+    "TargetRegion": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the region."
+        },
+        "regionalReplicaCount": {
```

Comment:
If the name of the region is called ```name```, why the replica count still has the ```region``` as the property's prefix? Why not just ```replicaCount```?

Reply:
Yes. Initially, we call it "replicaCount", the same as you suggested. Then we got some feedback from Kay Singh that it might be clearer to use "regionalReplicaCount". So we use this.

Reply:
OK